tools/firely-server-import.rst

```diff
@@ -0,0 +1,182 @@
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --indexBuffer <indexBuffer>               | workflow/indexBufferSize         |          | Number of resources to buffer for indexing the search parameters.                                                                          |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --version                                 |                                  |          | Show version information                                                                                                                   |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| -?, -h, --help                            |                                  |          | Show help and usage information                                                                                                            |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+
+Examples
+--------
+
+Runs the import for files located in directory **/path/to/your/input/files** using license file **/path/to/your/license/fsi-license.json** targeting the database defined by the connection string. In case if a resource being imported already exists in the target database, skip it.
+
+.. code-block:: bash
+
+  dotnet fsi.exe \
+  -s ./path/to/your/input/files \
+  --license /path/to/your/license/fsi-license.json \
+  -c 'Initial Catalog=VonkData;Data Source=server.hostname,1433;User ID=username;Password=PaSSSword!' \
+  --update-existing-resources false 
+
+Same as above but if a resource being imported already exists in the target database, it gets updated. The old resource gets preserved as a historical record.
+
+.. code-block:: bash
+
+  dotnet fsi.exe \
+  -s ./path/to/your/input/files \
+  --license /path/to/your/license/fsi-license.json \
+  -c 'Initial Catalog=VonkData;Data Source=server.hostname,1433;User ID=username;Password=PaSSSword!'
+
+Monitoring
+----------
+
+Logs
+^^^^
+
+When importing the data, it is handy to have the logging enabled, as it would capture any issues if they occur. By default, the log messages are written both to the console window and to the log files in the ``%temp%`` directory.
+
+You can configure the log settings the same way as you do for Firely Server: :ref:`configure_log`. 
```

Comment:
The use of a structured log capturing tool like Seq is beneficial, as the log output is structured as well.

Reply:
I didn't want to mention Seq because I thought it would be too much for this article.